test/node.js

```diff
@@ -276,43 +264,77 @@ node.expectedFeeForTrsWithData = function (amount) {
 	return parseInt(node.fees.transactionFee) + parseInt(node.fees.dataFee);
 };
 
-// Returns a random username
+// Returns a random username of 16 characters
 node.randomUsername = function () {
-	var size = node.randomNumber(1, 16); // Min. username size is 1, Max. username size is 16
-	var username = '';
-	var possible = 'abcdefghijklmnopqrstuvwxyz0123456789!@$&_.';
+	var randomLetter = randomString.generate({
+		length: 1,
+		charset: 'alphabetic',
+		capitalization: 'lowercase'
+	});
+	var custom = 'abcdefghijklmnopqrstuvwxyz0123456789!@$&_.';
+	var username = randomString.generate({
+		length: node.randomNumber(1,15),
+		charset: custom
+	});
 
-	for (var i = 0; i < size; i++) {
-		username += possible.charAt(Math.floor(Math.random() * possible.length));
-	}
+	return randomLetter.concat(username);
+};
+
+// Returns a random delegate name of 20 characters
+node.randomDelegateName = function () {
+	var randomLetter = randomString.generate({
+		length: 1,
+		charset: 'alphabetic',
+		capitalization: 'lowercase'
+	});
+	var custom = 'abcdefghijklmnopqrstuvwxyz0123456789!@$&_.';
+	var username = randomString.generate({
+		length: node.randomNumber(1,19),
```

Comment:
space (`1, 19`)

Reply:
Please @4miners , next time add a commit yourself addressing this purposes. We don't have a proper Eslint configuration which defines a style.